src/shared/components/phase-banner/phase-banner.component.html

```diff
@@ -4,7 +4,11 @@
               {{type}}
           </strong>
           <span class="govuk-phase-banner__text">
-              This is a new service – your <a class="govuk-link" href="https://www.smartsurvey.co.uk/s/CCDSurvey/" target="_blank" rel="noopener noreferrer" aria-label="provide your feedback">feedback</a> will help us to improve it.
+              This is a new service – your
+              <a class="govuk-link" href="https://www.smartsurvey.co.uk/s/CCDSurvey/" target="_blank" rel="noopener noreferrer" aria-label="provide your feedback - opens in a new window">
```

Comment:
Should we declare https:// ? I think with a link like this we should just use

`href="//www.smartsurvey.co.uk/s/CCDSurvey"`

Reply:
have removed the protocol as suggested